lib/auth/auth.go

```diff
@@ -1137,6 +1143,117 @@ func (a *Server) GetMFAAuthenticateChallenge(user string, password []byte) (*MFA
 	return chal, nil
 }
 
+// CreateAuthenticationChallenge creates and returns challenges for a users MFA devices.
```

Comment:
Suggestion: I don't find the comments in RPC-implementing-methods important - the RPC documentation is what matters imo. Fine by me if you want to omit or do something like `// FooRPC implements AuthService.FooRPC`.

Same for others.